data/fetch_datasets.ipynb

```diff
@@ -0,0 +1,81 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fetch AdKDD challenge dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget \"http://go.criteo.net/criteo-ppml-challenge-adkdd21-dataset.zip\"\n",
+    "!unzip criteo-ppml-challenge-adkdd21-dataset.zip && rm criteo-ppml-challenge-adkdd21-dataset.zip\n",
```

Comment:
is it necessary to unzip all csv files ?
if using numpy it's straightforward to read gzipped files
(not a big deal though)

Reply:
No it's not, I thought it was cleaner to do it this way. Should we keep .csv.gz files to reduce file space consumption?